src/pages/PhoneNumberPage.tsx

```diff
@@ -2,27 +2,39 @@ import "../components/modal/modal.scss";
 import Button from "../components/Button";
 import PhoneNumber from "../components/PhoneNumber";
 import {useState} from "react";
-import {Link} from "react-router-dom";
-
-const defaultCountry = '375';
+import {Link, useNavigate} from "react-router-dom";
+import {DEFAULT_COUNTRY} from "../constants";
 
 const PhoneNumberPage = () => {
-  const [error, setError] = useState('');
-  const [phoneNumber, setPhoneNumber] = useState(defaultCountry);
+  const [error, setError] = useState('Phone number incorrect');
+  const [phoneNumber, setPhoneNumber] = useState(DEFAULT_COUNTRY);
+  const [isError, setIsError] = useState(false);
+  const navigate = useNavigate();
 
+  const sendPhoneNumber = () => {
+    if (!error) {
+      navigate('/users/sign_in/phone_number/code');
+    } else {
+      setIsError(!!error)
```

Comment:
else {
      setIsError(true)
      
      as you definately know here that !!error = true